cmd/tdexd/main.go

```diff
@@ -262,9 +263,13 @@ func NewGrpcService(
 	tradeSvc application.TradeService,
 	repoManager ports.RepoManager,
 ) (interfaces.Service, error) {
-	hostname, err := config.GetHostname()
-	if err != nil {
-		return nil, err
+	addr := fmt.Sprintf("localhost:%d", operatorSvcPort)
+	if len(operatorTLSExtraDomains) > 0 {
+		addr = operatorTLSExtraDomains[0]
+	} else if len(operatorTLSExtraIPs) > 0 {
+		addr = operatorTLSExtraIPs[0]
```

Comment:
operatorTLSExtraDomains and operatorTLSExtraIPs should only be used for self-signed cert generation imho